### ISSUE TITLE
az login fails with "NotImplementedError: APIVersion 2019-06-01 is not available" in azure-cli 2.6.0

### ISSUE THREAD

**Issue by arendkolk** [Azure/azure-cli#16373]
## Describe the bug

**Command Name**
`az login`

**Errors:**
```
APIVersion 2019-06-01 is not available
Traceback (most recent call last):
python3.8/site-packages/knack/cli.py, ln 215, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 654, in execute
    raise ex
cli/core/commands/__init__.py, ln 718, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
cli/core/commands/__init__.py, ln 711, in _run_job
    six.reraise(*sys.exc_info())
...
azure/cli/core/_profile.py, ln 856, in find_through_interactive_flow
    result = self._find_using_common_tenant(token_entry[_ACCESS_TOKEN], resource)
azure/cli/core/_profile.py, ln 889, in _find_using_common_tenant
    tenants = client.tenants.list()
mgmt/resource/subscriptions/_subscription_client.py, ln 127, in tenants
    raise NotImplementedError("APIVersion {} is not available".format(api_version))
NotImplementedError: APIVersion 2019-06-01 is not available
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az login`

## Environment Summary
```
Linux-5.4.0-58-generic-x86_64-with-glibc2.29
Python 3.8.5
Installer: PIP

azure-cli 2.6.0

Extensions:
azure-devops 0.17.0

```
## Additional Context

EXTRA information:
arend@ASUS-IntelCoreI7:~/Documents/azure/tutorial2$ az --version
azure-cli                          2.6.0

command-modules-nspkg              2.0.3
core                               2.6.0
nspkg                              3.0.4
telemetry                          1.0.6

Extensions:
azure-devops                      0.17.0

Python location '/usr/bin/python3'
Extensions directory '/home/arend/.azure/cliextensions'
Extensions system directory '/usr/lib/python3/dist-packages/azure-cli-extensions'

Python (Linux) 3.8.5 (default, Jul 28 2020, 12:59:40) 
[GCC 9.3.0]

**Comment by yonzhan** [comment]
az login

**Comment by arendkolk** [comment]
i changed Ubuntu back from version 20 to version 18.04
no more problems, also i installed azure-cli 2.6 without problems.

**Comment by arendkolk** [comment]
installed azure-cli 2.6 via "pip3 install azure-cli==2.6"

2.6 was needed for the [tutorial](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-tutorial-create-first-template?tabs=azure-cli)

**Comment by jiasli** [comment]
Hi @arendkolk, it looks like you are hitting a similar issue as #12779 which we wasn't able to find the root cause at last. Could you check #12779 and share the result of

```
> pip list | grep azure-mgmt-resource

azure-mgmt-resource                   8.0.1
```

By the way, [this tutorial](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-tutorial-create-first-template) suggests to use **2.6 or later**. 

> If you're using Azure CLI, make sure you have version 2.6 or later. The commands shown in this tutorial will not work if you're using earlier versions.

You may update to the latest version and see if the issue persists.

**Comment by jiasli** [comment]
Another occurrence of this `NotImplementedError: APIVersion 2019-06-01 is not available` error is on OpenSUSE (#13209), but as you are using Ubuntu, these 2 issues could possibly not be are relevant.

**Comment by jiasli** [comment]
Closing due to no response.
